x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/ingest/CircleProcessorTests.java

```diff
@@ -60,22 +58,6 @@
 
 public class CircleProcessorTests extends ESTestCase {
 
-    public void testNumSides() {
-        double radiusDistanceMeters = randomDoubleBetween(0.01, 6371000, true);
-        CircleShapeFieldType shapeType = randomFrom(SHAPE, GEO_SHAPE);
-        CircleProcessor processor = new CircleProcessor("tag", null, "field", "field", false, radiusDistanceMeters, shapeType);
-
-        // radius is same as error distance
-        assertThat(processor.numSides(radiusDistanceMeters), equalTo(4));
-        // radius is much smaller than error distance
-        assertThat(processor.numSides(0), equalTo(4));
-        // radius is much larger than error distance
-        assertThat(processor.numSides(Math.pow(radiusDistanceMeters, 100)), equalTo(1000));
```

Comment:
The previous test is flaky. If `radiusDistanceMeters` is `1.11` or less then this assert won't pass because when we power the number in 100 it won't be big enough. Moreover, if the number is less then 1 then power will work in another direction which will make `radiusDistanceMeters` smaller then the error distance used.
I'm will fix it in the test

Reply:
@iverase fixed the test. It's now called `testCircleToPolygonNumSides`. In addition to testing random values between 0.01 and 6371000 it's now testing fixed small and big values as well to avoid flakiness.